### PR TITLE
Add thailand logo

### LINKED ISSUE #843: RVL-936 Change logo on Reveal and Superset Thailand
We need to change the logo to the Thailand instance of the web UI. The logo placement is in the issue. Please work with Pedro if you need a higher fidelity image from the BVBD.

https://smartregister.atlassian.net/browse/RVL-936

### PR DESCRIPTION
**Pull request by p-netm**
close #843
Add thailand logo conditionally
Adds an env REACT_APP_PRODUCTION_INST that tells the client instance.

### CODE REVIEW

**Comment by moshthepitt** [comment]
@p-netm where is this being added?

**Comment by p-netm** [comment]
> @p-netm where is this being added?

@moshthepitt the logo you mean?

**Comment by moshthepitt** [comment]
Yeah, where does the logo appear?

Also, please read this thread: https://onaio.slack.com/archives/CBSU8L2SC/p1589375814364900

**Comment by p-netm** [comment]
> Yeah, where does the logo appear?
> 
> Also, please read this thread: https://onaio.slack.com/archives/CBSU8L2SC/p1589375814364900

Had not seen this.

**Comment by p-netm** [comment]
@moshthepitt Not quite sure what's the take on this, seems like scoping is yet to be completed.

**Comment by moshthepitt** [comment]
Yes, we still lack sufficient scoping @p-netm

**Comment by p-netm** [comment]
Screenshot from 2020-06-03 14-49-07
Screenshot from 2020-06-03 14-48-58

File: src/components/page/Header/index.tsx [hunk 1]
```diff
@@ -100,9 +103,15 @@ export class HeaderComponent extends React.Component<HeaderProps, State> {
...
-            </Link>
+            {REACT_APP_PRODUCTION_INST === TH_PRODUCTION_INST ? (
+              <Link to="/" className="navbar-brand">
+                <img src={thaiLogo} alt={WEBSITE_NAME} style={{ transform: 'scale(2)' }} />
+              </Link>
+            ) : (
+              <Link to="/" className="navbar-brand">
+                <img src={logo} alt={WEBSITE_NAME} />
+              </Link>
+            )}
```
Comment (moshthepitt) [inline src/components/page/Header/index.tsx:114]: How do you compare doing it this way Vs having the value of the `img` `src` come from a config?

File: src/components/page/Footer/footer.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+.reveal-floating-logo img {
+  width: 100px !important;
+  height: auto;
+}
+
+.floating-logo-text {
+  vertical-align: bottom;
+  line-height: 45px;
+}
```
Comment (moshthepitt) [inline src/components/page/Footer/footer.css:15]: Was there a requirement to make this thing float around?  I don't like that it does, and it just makes the page more busy without adding value other than "hey see our nice shiny logo".

File: src/components/page/Header/index.tsx (hunk 1 above)
Comment (p-netm) [reply src/components/page/Header/index.tsx:114]: yeah I think using the src is more flexible.

**Comment by p-netm** [comment]
```
# REACT_APP_NAVBAR_BRAND_IMG_SRC=https://github.com/onaio/reveal-frontend/raw/b0756b0c1ed95dc6db23dc4fd1375c81bdf7fb1b/src/assets/images/logo.png
REACT_APP_NAVBAR_BRAND_IMG_SRC=https://github.com/onaio/reveal-frontend/raw/aef6d6410a79d5f3c4535e3269ba72bca1eac674/src/assets/images/thai-logo.png
REACT_APP_REVEAL_BRAND_IMG_SRC=https://github.com/onaio/reveal-frontend/raw/aef6d6410a79d5f3c4535e3269ba72bca1eac674/src/assets/images/logo-small.png

```

**Comment by moshthepitt** [comment]
@p-netm could you post screenshots?  I am not able to see the changes on my side

**Comment by p-netm** [comment]
Screenshot from 2020-06-05 20-14-49
Screenshot from 2020-06-05 20-15-39
@moshthepitt

**Pull request merged**